lib/client_side_validations/simple_form/form_builder.rb

```diff
@@ -28,12 +28,22 @@ def input(attribute_name, options = {}, &block)
       private
 
       def wrapper_error_component
-        if wrapper.components.map(&:namespace).include?(:error)
+        if namespace_present?(wrapper, :error)
           wrapper.find(:error)
         else
           wrapper.find(:full_error)
         end
       end
+
+      def namespace_present?(component, namespace)
```

Comment:
I really like the approach you suggested @tagliala. It's much cleaner, and less reliant on `simple_form`-specific classes. I modified your suggestion a bit by generalizing `#namespace_present?`, to provide some flexibility if needed later on.

Let me know your thoughts about this.